e20-kit3-demo/snapconnect-exosite-example/main.py

```diff
@@ -0,0 +1,100 @@
+import os
+import sys
+import json
+import binascii
+
+from snapconnect import snap
+from apy import ioloop_scheduler
+
+import tornado.ioloop
+from tornado import httpclient
+
+from pyonep import onep  # Exosite Python Library
+
+
+# TODO: Replace these with values from your own exosite account and resource
+# We like to map SNAP addresses to Exosite CIKs
+EXOSITE_CIKS = {"000020": ''}
```

Comment:
I was debating if we should show a CIK since it's just a UUID.

As for the MAC address, it should be a node MAC address not SNAP Connect. I left 000020 in there from when I was testing with SNAP Connect only.